infra/helper.py

```diff
@@ -419,6 +434,33 @@ def check_build(args):
   return exit_code
 
 
+def profile(args):
+  """Generate code coverage using clang source based code coverage."""
+  if not _check_project_exists(args.project_name):
+    return 1
+
+  env = [
+      'FUZZING_ENGINE=libfuzzer',
+      'PROJECT=%s' % args.project_name,
+      'SANITIZER=profile'
+  ]
+
+  run_args = _env_to_docker_args(env) + [
+      '-v', '%s:/out' % os.path.join(BUILD_DIR, 'out', args.project_name),
+      '-p', '8008:8008',
+      '-t', 'gcr.io/oss-fuzz/%s' % args.project_name,
```

Comment:
Why not do this in the runner? Since we already copy sources to /out

Reply:
I think I've just copied that behavior from existing `coverage()` helper which generates the report inside builder container. I suspect there might be a little advantage, if user opens `shell`, does something inside the image and wants to re-run the coverage report generation.

Reply:
Moved to the runner though.